src/main/java/org/generationcp/middleware/api/breedingmethod/BreedingMethodServiceImpl.java

```diff
@@ -67,6 +68,12 @@ public BreedingMethodDTO getBreedingMethod(final Integer breedingMethodDbId) {
 			.collect(Collectors.toList());
 	}
 
+	@Override
+	public Integer countBreedingMethods(final BreedingMethodSearchRequest methodSearchRequest) {
+		final List<BreedingMethodDTO> breedingMethodDTO = this.getBreedingMethods(methodSearchRequest);
```

Comment:
I would prefer we don't use `getBreedingMethods` then count but instead implement a `countBreedingMethods `in `MethodDao`. You can find an example of the filtered getXX and countXX in `LocationDao#countFilterLocations`.